tests/test_playbooks/tasks/lifecycle_environment.yml

```
@@ -13,6 +13,7 @@
     name: "{{ lifecycle_environment_name }}"
     label: "{{ lifecycle_environment_label }}"
     organization: "{{ organization_name }}"
+    description: "{{ lifecycle_environment_description }}"
```

```suggestion
    description: "{{ lifecycle_environment_description | default(omit) }}"
```

as the description is optional